vllm/inputs/registry.py

```diff
@@ -65,6 +66,9 @@ def get_hf_image_processor_config(self) -> Dict[str, Any]:
 
 N = TypeVar("N", bound=Type[nn.Module])
 
+DummyDataTuple = Tuple["SequenceData", Optional["MultiModalDataDict"],
+                       Optional["MultiModalPlaceholderDict"]]
```

Comment:
Make this a `NamedTuple` or `TypedDict` to make the code clearer?

Reply:
Good suggestion, much better.